README.markdown

```diff
@@ -28,7 +28,7 @@ Installation
 
 Install Redigo using the "go get" command:
 
-    go get github.com/gomodule/redigo/redis
+    go get -u github.com/gomodule/redigo/redis/v3
 
 The Go distribution is Redigo's only dependency.
 
```

Comment:
Given we're making a change which will start erroring when people try to update their dependencies I think we should add a upgrading section, to help people on the their way. I would suggest something like the following which is based off the FAQ entry:

Updating
----------
If you you were using `redigo` prior to the `v3` tag and are using `go mod` you'll need to update your import URL's to add the `/v3` suffix as detailed by [Installation](#installation). A community tool [github.com/marwan-at-work/mod](https://github.com/marwan-at-work/mod) helps automate this procedure. See the [repository](https://github.com/marwan-at-work/mod) or the [community tooling FAQ](https://github.com/golang/go/wiki/Modules#what-community-tooling-exists-for-working-with-modules) below for an overview.

Thoughts?

Reply:
[I added it](https://github.com/dcormier/redigo/blob/dc/gomod/README.markdown#updating). I changed "import URL's" to "import paths", but it's otherwise untouched.